src/eva/transforms/channel_stats.py

```diff
@@ -0,0 +1,66 @@
+# (C) Copyright 2021-2022 NOAA/NWS/EMC
+#
+# (C) Copyright 2021-2022 United States Government as represented by the Administrator of the
+# National Aeronautics and Space Administration. All Rights Reserved.
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+
+# --------------------------------------------------------------------------------------------------
+
+
+from math import sqrt
+import re
+from statistics import mean
+
+from eva.utilities.config import get
+from eva.utilities.logger import Logger
+from eva.transforms.transform_utils import parse_for_dict, split_collectiongroupvariable
+from eva.transforms.transform_utils import replace_cgv
+from eva.utilities.utils import remove_list_duplicates
+from eva.utilities.utils import remove_empty_from_list_of_strings
+
+
+# --------------------------------------------------------------------------------------------------
+
+
+def channel_stats(config, data_collections):
+
+    # Create a logger
+    logger = Logger('ChannelStatsTransform')
+
+    # Define default stat functions to calculate
+    if 'statistic list' in config:
+        stat_functions = get(config, logger, 'statistic list')
```

Comment:
Last comment and then I'll leave @ADCollard alone. Do we want to check if the input list is 'valid'? Either here but verifying that the list contains valid values. Or I suggest on line ~59, should have a try/except or some sort of check so it doesn't fail completely? (Or do we want it to fail completely?)

Reply:
@CoryMartin-NOAA I think it should fail completely.  You want to be sure your YAML is right and does what you want rather than just ignoring all your mistakes.

I guess I can add some code to make it fail more elegantly, but inscrutable messages when you get the YAML wrong seems to be the norm as far as I can tell.  :-)

Reply:
Roger that.